docs/governance.rst

```diff
@@ -0,0 +1,244 @@
+   Issue.
+2. Every modification of the specification (including a correction of a typo,
+   adding a new Contributor, an extension adding support for a new data type,
+   or others) or proposal to release a new version needs to be done via a
+   Pull Request (PR) to the Repository.
+3. Anyone can open an Issue or a PR (this action is not limited to Contributors).
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR of which neither is the author of the PR. 
+      The reviews should be made after the last commit in the PR (equivalent to 
+      `Stale review dismissal <https://help.github.com/articles/enabling-required-reviews-for-pull-requests/>`_
+      option on GitHub). If a second review requests minor changes after
+      another reviewer approved the PR, the first review does not need
+      to re-review.
+   b) Does not feature any `Reviews that Request changes <https://help.github.com/articles/about-required-reviews-for-pull-requests/>`_.
+      That is, if someone asked for changes, the PR should not be merged just
+      because two other people approve it.
+   c) Is not a Draft PR. That is the PR author says it is ready for review.
+   d) Passes all automated tests.
+   e) Is not proposing a new release
+   f) The steering committee has not added extra restrictions. For example, if
+      a PR is a non-trival change, the steering committee can create a system
+      to get feedback from more than just two reviewers before merging.
+5. After consultation with contributors, the steering committee can decide
+   to merge any PR - even if it's not eligible to merge according to Rule 4.
+6. Anyone can Review a PR and request changes. If a community
+   member Requests changes they need to provide an explanation regarding what
+   changes should be made and justification of their importance. Reviews
+   requesting changes can also be used to request more time to review a PR.
+7. A reviewer who Requested changes can dismiss their own review, if
+   they decide their requested changes are no longer necessary, or approve
+   changes that address the issue underlying their change request.
+8. If the author of a PR and a reviewer who requests changes cannot find a
+   solution that would lead to: (1) The author closing the PR without merging
+   (2) The reviewer accepting requested changes or (3) The dismissing their
+   review, so that the PR can be approved and merged, then the disagreement
+   will be resolved with a vote.
+9. Rules governing voting:
+
+   a) A vote can be triggered by any Maintainer, but only after 5 working days
+      from the time a Review Requesting Changes. If a PR has a disagreement
+      that required a vote, there must be at least 15 days from the conclusion
+      of the first vote before another vote can be triggered.
```

Comment:
```suggestion
      from the time a Review Requesting Changes. A PR can only have one 
      open vote at a time. If disagreements over a PR results in more than one
      vote, the Steering Committee has the authority to  create a voting process
      to help resolve disagreements in a more efficient and respectful manner.
```

Reply:
@emdupre & @jbteves I did propose this change based on several comments. I removed the 15 day delay and simply said we should never have more than one vote on a PR happening at one time and that, if a PR goes through more than one vote, the steering committee can decide how to create a resolution process. I think this get's at Elizabeth's point in her comment that, if there are repeated votes, the issue is due to the scope or explanation rather than the specific detail at hand. I think this is better than the arbitrary, but specific 15-days rule.
Does this look like a reasonable change to both of you?

Reply:
I prefer this version, and think it's good.